hack/pedantic-imports.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+
+# Copyright 2019 The Skaffold Authors
```

Comment:
2020

Reply:
great catch - I still owe with the updated script :D